tools/packaging/kata-deploy/scripts/kata-deploy.sh

```diff
@@ -108,8 +110,11 @@ function install_artifacts() {
 	echo "copying kata artifacts onto host"
 	cp -au /opt/kata-artifacts/opt/kata/* /opt/kata/
 	chmod +x /opt/kata/bin/*
-	[ -d /opt/kata/runtime-rs/bin ] && \
+	[ "$SELINUX" == "yes" ] && chcon -Rt usr_t /opt/kata/ && chcon -t bin_t /opt/kata/bin/*
```

Comment:
Can you also add a print here to say "SELINUX is being used" or something similar?

I think it's also worth adding a test on a system which has SELINUX enforced and ensure which hypervisors will work with that ... but that's material for a follow-up PR.

Reply:
Actually currently we have `selinux` disabled in the periodic https://prow.ci.openshift.org/job-history/gs/origin-ci-test/logs/periodic-ci-kata-containers-kata-containers-main-e2e-tests test, with this we might be able to enable it (haven't tested it yet, though). Current motivation is to help me with my kcli deployment based on centos, which suffers the selinux issues.

So would enabling it in the periodic ocp test suffice or you have a lower-level test in mind?